frontend/src/views/GNewShoot.vue

```diff
@@ -512,8 +512,8 @@ export default {
       const end = get(shootResource, 'spec.maintenance.timeWindow.end')
       const k8sUpdates = get(shootResource, 'spec.maintenance.autoUpdate.kubernetesVersion', true)
       const osUpdates = get(shootResource, 'spec.maintenance.autoUpdate.machineImageVersion', true)
-      this.$refs.maintenanceTime.setBeginTimeTimezoneString(begin)
-      this.$refs.maintenanceTime.setEndTimeTimezoneString(end)
+      this.$refs.maintenanceTime.setBeginTimeAndTimezone(begin)
+      this.$refs.maintenanceTime.setWindowDuration(end)
```

Comment:
```suggestion
      this.$refs.maintenanceTime.setMaintenanceWindow(begin, end)
```

I think we should have one method 'setMaintenanceWindow' that changes the maintenance window. This would also make clear what is does when called from outside, I think. The implementation could look like:
```js
    reset () {
      this.setMaintenanceWindow(this.timeWindowBegin, this.timeWindowEnd)
    },
    ...
    setMaintenanceWindow (begin, end) {
      const defaultDuration = 60
      if (begin && end) {
        const beginTime = new TimeWithOffset(begin)
        if (beginTime.isValid()) {
          this.maintenanceBegin = beginTime.getTimeString()
          this.maintenanceTimezone = beginTime.getTimezoneString()
        }
        const endTime = new TimeWithOffset(end)
        if (endTime.isValid()) {
          const duration = getDurationInMinutes(this.maintenanceBegin, endTime.getTimeString())
          this.windowDuration = duration > 0 ? duration : defaultDuration
        }
      } else {
        this.maintenanceBegin = randomMaintenanceBegin()
        this.maintenanceTimezone = this.timezone
        this.windowDuration = defaultDuration
      }
    },
    ...
```